airbyte-integrations/connectors/source-mongodb-v2/build.gradle

```diff
@@ -5,9 +5,9 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.1.2'
+    cdkVersionRequired = '0.1.13'
     features = ['db-sources']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
I don't think that we want to commit this change, but I could be wrong.  When I released this originally, I was told to commit the changes to the CDK first and release it via CI slash commands, then do the update to the connector itself to consume the newly published update.

Reply:
true true. putting back to false
thanks